connexion/decorators/validation.py

```diff
@@ -89,6 +89,11 @@ def extend_with_nullable_support(validator_class):
     def nullable_support(validator, properties, instance, schema):
         null_properties = {}
         for property_, subschema in six.iteritems(properties):
+            if "$ref" in subschema:
+                ref_schema = validator.resolver.resolve(subschema["$ref"])
+                if len(ref_schema) == 2:
```

Comment:
what does this do?
can you remove the magic number / add a comment?

Reply:
`ref_schema` returns a tuple like so like the $ref and the actual schema definition ("definitions/foo", {}). Also yeh I can do that if you want.

Reply:
I had done the review before realizing this was against v1_unsupported.
I don't have a strong preference.

Another way:
```
try:
    _, subschema = validator.resolver.resolve(subschema["$ref"])
except ValueError:
    pass
```

Reply:
Yes that is much cleaner